tests/components/wyoming/test_tts.py

```diff
@@ -0,0 +1,131 @@
+"""Test tts."""
+from __future__ import annotations
+
+import io
+from unittest.mock import patch
+import wave
+
+from wyoming.audio import AudioChunk, AudioStop
+
+from homeassistant.components import tts
+from homeassistant.core import HomeAssistant
+
+from . import MockAsyncTcpClient
+
+
+async def test_support(hass: HomeAssistant, init_wyoming_tts) -> None:
+    """Test supported properties."""
+    state = hass.states.get("tts.wyoming")
+    assert state is not None
+
+    entity = tts.async_get_text_to_speech_entity(hass, "tts.wyoming")
```

Comment:
We normally don't assert entity properties directly, but the state of the entity. We could consider adding the entity features to the state attributes if they are important for automations.

Reply:
Yeah we need to come up with a solution for this. Right now it was copied from STT but we're trying to remove that one as well.

Reply:
The difference compared to state-entities here is that these properties don't impact the state of the entity but instead describe the type of input it can handle for it's service.